tests/cli/cli_test.go

```diff
@@ -127,6 +127,14 @@ func writeNetrc(token, uri string) (*os.File, error) {
 	return f, nil
 }
 
+func deleteNetrc() error {
```

Comment:
Ah, that makes sense. I wonder if it would be more straight forward to just set `NETRC_PATH` in NewCmd. Then have the `Authorize` method just write to that file.